fe/fe-core/src/main/java/com/starrocks/scheduler/TaskBuilder.java

```diff
@@ -28,4 +33,19 @@ public static Task buildTask(SubmitTaskStmt submitTaskStmt, ConnectContext conte
         return task;
     }
 
+    public static Task buildMvTask(MaterializedView materializedView, String dbName) {
+        Task task = new Task();
+        long taskId = GlobalStateMgr.getCurrentState().getNextId();
+        task.setId(taskId);
+        task.setName("mv-" + materializedView.getId());
+        task.setSource(Task.TaskSource.MV);
+        task.setCreateTime(System.currentTimeMillis());
+        task.setDbName(dbName);
+        Map<String, String> taskProperties = Maps.newHashMap();
+        taskProperties.put(MvTaskRunProcessor.MV_ID, String.valueOf(materializedView.getId()));
+        task.setProperties(taskProperties);
+        task.setDefinition(materializedView.getViewDefineSql());
+        task.setExpireTime(System.currentTimeMillis() + Config.task_ttl_second * 1000L);
```

Comment:
here should set expireTime to 0, because the life cycle of Task should be handed over to MV management, not the Task framework